zerver/tests/test_import_export.py

```diff
@@ -127,7 +127,9 @@ def get_huddle_hashes(r: Realm) -> str:
     return huddle_hash
 
 
-class RealmImportExportTest(ZulipTestCase):
+class ExportFile(ZulipTestCase):
```

Comment:
Being a subclass of `TestCase`, I think the test runner will try to run this class.  I guess this is OK since unlike `SocialAuthBase`, this doesn't have any actual `test_` functions attached to it, so it'll run 0 tests when doing so.